.github/workflows/cross_compile.yml

```diff
@@ -17,11 +17,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: stable
+          toolchain: nightly
```

Comment:
As mentioned on slack, we should probably stay on stable for CI to avoid accidental drift into nightly-required.